### PR TITLE
Basic tools for parsing GA data

### PR DESCRIPTION
**Pull request by betatim**
This is the code I used to produce the data mentioned in #12

### CODE REVIEW

**Comment by betatim** [comment]
If people are happy with the level of summarisation done I will add the actual data files to this PR before merging.

**Comment by choldgraf** [comment]
cool! FWIW, I've been using this repo *just* for data and scripts to update data, and was planning to use other repositories for analyzing/visualizing/etc. E.g. the billing data exists here at `billing/` but all the analysis scripts live at https://github.com/jupyterhub/binder-billing. Do you think this is too convoluted? I am 50/50

**Comment by betatim** [comment]
Yes, these notebooks only munge the data from its raw form into the "public" version.

**Comment by choldgraf** [comment]
yes you think it's too convoluted?

**Comment by betatim** [comment]
No. I like the split between data preprocessing and doing stuff with it.

Now I am confused though. I read your original comment as "this PR contains stuff that isn't just data preprocessing and should be in a different repo" which is why I replied "this is just preprocessing".

The notebooks in this PR preprocess the data. The daily visits one also makes a quick plot of the data to verify the munging worked before writing it to a CSV.

File: traffic/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## How to update the per repo and per URL data
+
+1. Visit [Google Analytics](https://analytics.google.com/analytics/web/#/report/content-pages/a101904940w149250546p154152886/_u.dateOption=last30days&explorer-table.plotKeys=%5B%5D&explorer-table.rowCount=5000)
+2. Check that the last 30 days of data are shown and that it is no more than 5000 rows
+3. Export the data to a CSV
+4. Adjust the file name used in `summarize-data.ipynb`
+5. Run the `summarize-data.ipynb` notebook to produce the file to commit to
+   this repository.
+6. Commit the resulting `page-20180820-20180918.csv` and
+   `repo-20180820-20180918.csv` files that cover the 30 period you extracted.
```
Comment (betatim) [inline traffic/README.md]:
```suggestion
   `repo-20180820-20180918.csv` files that cover the 30 day period you extracted.
```

**Comment by yuvipanda** [comment]
5 years late is better than never <3

**Pull request merged**